src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
```

Comment:
why not `try-with-resources`?

Reply:
Because in checklist we have such line:
Use try with finally, where you use transaction and try with resources with Read operations.

Reply:
@Vt0rtikV that's a really weird "mistake" :)

there is no problem at all with using `try-with-resources` here

but let it be :) try-finally neither breaks anything